src/System/Requirement/InstallationNotOverriden.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2022 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\System\Requirement;
+
+use Config;
+use DBmysql;
+use FilesystemIterator;
+use Glpi\Toolbox\VersionParser;
+
+final class InstallationNotOverriden extends AbstractRequirement
+{
+    /**
+     * Database instance.
+     *
+     * @var DBmysql
+     */
+    private $db;
+
+    /**
+     * Version directory.
+     *
+     * @var string
+     */
+    private $version_dir;
+
+    public function __construct(?DBmysql $db, string $version_dir = GLPI_ROOT . '/.version')
+    {
+        $this->db = $db;
+        $this->version_dir = $version_dir;
+
+        $this->title = __('Anterior versions files detection');
```

Comment:
I'm not sure about using "Anterior versions" here. Maybe "Previous GLPI version" instead would be better?

Reply:
```suggestion
        $this->title = __('Previous GLPI version files detection');
```